### PR TITLE
add Chinese localization for the awesome list

### PR DESCRIPTION
**Pull request by SangKa**
Hi @mattiamanzati @hawkins , I maintain the [Chinese localization for the MobX Docs](https://github.com/mobxjs/mobx#translations).

Now the rescource section of docs has removed into the [Awesome MobX](https://github.com/mobxjs/awesome-mobx). For the syncing with offical docs, I create the `README-CN` and language switch link both in the two `README`. Hope this can help more MobXer which come from China.

Lastly, I will maintain the `README-CN` continuely to keep in sync with you guys.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 A collection of awesome things regarding MobX.
 
+English | [简体中文](README-CN.md)
```
Comment (hawkins) [inline README.md]: Should this instead read "Chinese"?

**Comment by hawkins** [comment]
Sounds great to me, thanks a ton! 👍 😄 

Is there anything we can do to help make translation easy?

**Comment by SangKa** [comment]
@hawkins , no thanks. I will track each change and keep in sync with you.

**Comment by hawkins** [comment]
Sounds good to me. 👍

**Pull request merged**